src/HTTP.jl

```diff
@@ -347,7 +350,7 @@ After a suspected connection task failure:
 """
 function serve(serverws::ServerWS, host, port, verbose)
     # An internal reference used for closing.
-    tcpserver = Ref{Union{IOServer, Nothing}}(Sockets.listen(InetAddr(parse(IPAddr, host), port)))
```

Comment:
It would be nice if
```
IPv4(host::IPv4) 
```
existed in Sockets, just like 
```
Int(i::Int) 
```
exists. But it doesn't, just like ServerWS(s::ServerWS) doesn't exist.